pom.xml

```diff
@@ -41,14 +41,15 @@
     <maven.compiler.version>1.8</maven.compiler.version>
     <spring.version>5.2.6.RELEASE</spring.version>
     <spring.boot.version>2.3.3.RELEASE</spring.boot.version>
+    <slf4j.version>1.7.30</slf4j.version>
+    <jackson.version>2.11.2</jackson.version>
     <!-- metadb messaging and shared entities dependency versions -->
-    <cmo_metadb_messaging_java.group>com.github.mskcc</cmo_metadb_messaging_java.group>
-    <cmo_metadb_messaging_java.version>39241a484410204dd27112e074578d887323b801</cmo_metadb_messaging_java.version>
```

Comment:
TODO: Replace commit and group with cmo-messaging-java upstream/master after merging: https://github.com/mskcc/cmo-messaging-java/pull/14/files

Reply:
done